saleor/graphql/shop/mutations/shop_domain_update.py

```diff
@@ -24,7 +25,7 @@ class Arguments:
         input = SiteDomainInput(description="Fields required to update site.")
 
     class Meta:
-        description = "Updates site domain of the shop."
+        description = "Updates site domain of the shop." + DEPRECATED_IN_3X_MUTATION
```

Comment:
Would be worth mentioning how you can set it up now

Reply:
Great point! Done